src/test/resources/jsonpatch/extended/testsuite.json

```diff
@@ -11,6 +11,12 @@
         "patch": [{"op": "omit?", "path": "/foo", "value": "bar"}],
         "expected": {}
     },
+  {
+        "comment": "translate operation is able to be parsed and applied",
+        "doc": {"foo": "bar"},
+        "patch": [{"op": "trasnlate", "path": "/foo", "from": "bar", "value": "qux"}],
```

Comment:
It looks like if the op itself is invalid, it skips this test. Maybe exception thrown in the data provider execution? Not sure how the internals of ObjectReader.readValue(...) works on line 64 of JsonPatchTestSuite.java

The entire file in general is being executed though